.github/ISSUE_TEMPLATE/Bug_Report.yml

```diff
@@ -0,0 +1,124 @@
+name: Bug report 🐛
+description: Report errors or unexpected behavior 🤔
+labels:
+  - Issue-Bug
+  - Need-Review
+body:
+- type: checkboxes
+  attributes:
+    label: Prerequisites
+    options:
+    - label: Write a descriptive title.
+      required: true
+    - label: Make sure you are able to repro it on the latest version
+      required: true
+    - label: Search the existing issues.
+      required: true
+- type: textarea
+  attributes:
+    label: Summary
+    description: >-
+      Write a short description of the issue at a high-level.
+    placeholder: >-
+      I am experiencing a problem with X.
+      I think Y should be happening but Z is actually happening.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps to reproduce
+    description: >
+      List of steps, sample code, failing test or link to a project that reproduces the behavior.
+      Make sure you place a stack trace inside a code (```) block to avoid linking unrelated issues.
+    placeholder: |-
+      1. Create the following configuration document:
+
+         ```yaml
+         # repro.dsc.config.yaml
+         $schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/config/document.json
+         resources:
+         - name: repro
+           type: Test/Echo
+           properties:
+             output: expected value
+         ```
+
+      1. Call the `get` operation on the repro document:
+
+         ```sh
+         dsc config get --path ./repro.dsc.config.yaml
+         ```
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected behavior
+    render: console
+    placeholder: |
+      PS> dsc config get --path ./repro.dsc.config.yaml
+      
+      results:
+      - name: repro
+        type: Test/Echo
+        result:
+          actualState:
+            output: expected value
+      messages: []
+      hadErrors: false
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Actual behavior
+    render: console
+    placeholder: |
+      PS> dsc config get --path ./repro.dsc.config.yaml
```

Comment:
We should ask to have `debug` tracing enabled as it shouldn't leak any private data:

```powershell
# enable debug tracing which should not contain any private data
dsc -l debug config get ...`
```